src/json-uri.cpp

```diff
@@ -24,164 +24,165 @@
  * THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 #include "json-schema.hpp"
+#include <sstream>
 
 namespace nlohmann
 {
 
 void local_json_pointer::from_string(const std::string &r)
 {
-	str_ = "#";
+  str_ = "#";
 
```

Comment:
Could you re-create a commit (and push-force) without all these white-space-changes?

Reply:
sure. already done.